cg/cli/demultiplex/demux.py

```diff
@@ -170,7 +177,8 @@ def delete_flow_cell(
         )
 
 
-@click.command(name="copy-novaseqx")
+@click.command(name="copy-completed")
+@click.pass_obj
 def copy_novaseqx_flow_cells(context: CGConfig):
     """Copy novaseqx flow cells ready for post processing to demultiplexed runs."""
     flow_cells: Path = context.demultiplex.out_dir

```

Comment:
```suggestion
    flow_cells_directory: Path = context.flow_cells_dir
    demultiplexed_runs_directory: Path = context.demultiplexed_flow_cells_dir
```
I think we want to copy it from `flow_cells_dir` to `demultiplexed-runs`. Or where does it get the `proj/production/flow_cells/` directory from?
Also the `demultiplex.out_dir` (i.e. demultiplexed_runs) can be accessed via `context.demultiplexed_flow_cells_dir` I think these two got mixed up here. The new naming of the directories should be clearer.

Reply:
Yeah, that is what I intend (copy it from flow_cells_dir to demultiplexed-runs). Double checking it

Reply:
Fixed 👍